### PR TITLE
:bar_chart: Combined origin in population table

### PR DESCRIPTION
**Pull request by Marigold**
Implementation of https://github.com/owid/etl/issues/1451

* Have two tables `population_original` and `population` with the same data, but collapsed origin
* Use `population_original` in grapher population and garden population density (population density isn't reused by other datasets, so expanded origins in garden step are ok)
* [Collapsed Origin](https://github.com/owid/etl/compare/refactor-population...1451-population-dataset-condensed?expand=1#diff-f9d2ff0d8e86c9309ad0676931c890a21a88672ba19cd51ae925c30480b55d3cR1) could be improved for sure

### CODE REVIEW

File: etl/steps/data/garden/demography/2023-03-31/population/population.meta.yml
Comment (pabloarosado) [inline etl/steps/data/garden/demography/2023-03-31/population/population.meta.yml]: `collapsed_origin` sound like a Christopher Nolan movie. Shall we call it `origin_combined`? If it's annoying, leave it like that.

**Review by pabloarosado — APPROVED** [review]
Thanks, it looks good! Before merging, let me check how it looks on different cases, to see if we should tweak the new metadata a bit. Otherwise, we can merge it and tweak it afterwards (I suppose the former option is better, given that it will trigger a huge ETL job).

**Comment by Marigold** [comment]
> Thanks, it looks good! Before merging, let me check how it looks on different cases, to see if we should tweak the new metadata a bit. Otherwise, we can merge it and tweak it afterwards (I suppose the former option is better, given that it will trigger a huge ETL job).

Agree. Once it is good to go, we should merge this PR into #1536 and then that one to trigger the rebuild.

**Comment by Marigold** [comment]
@pabloarosado I rebased again and updated all YAML files with the new metadata. Could you take a look at the [combined population origin metadata](https://github.com/owid/etl/pull/1528/files#diff-f9d2ff0d8e86c9309ad0676931c890a21a88672ba19cd51ae925c30480b55d3cR108) and let me know if it can be improved?

**Comment by Marigold** [comment]
There's an issue when combining sources and origins (e.g. [this chart](http://staging-site-mojmir/admin/charts/4466/edit)) - only origin's full citation in `Data published by` and sources are missing. Below is a version with only sources
image
and [here is](http://staging-site-mojmir/admin/charts/4466/edit) version with sources and origins
image

**Comment by Marigold** [comment]
@pabloarosado @lucasrodes here are charts using new population

## Population charts
http://staging-site-mojmir/grapher/population
http://staging-site-mojmir/grapher/population-density

## Charts using combined population (20 random)
http://staging-site-mojmir/admin/charts/4556/edit
http://staging-site-mojmir/admin/charts/4381/edit
http://staging-site-mojmir/admin/charts/4451/edit
http://staging-site-mojmir/admin/charts/6935/edit
http://staging-site-mojmir/admin/charts/6271/edit
http://staging-site-mojmir/admin/charts/4254/edit
http://staging-site-mojmir/admin/charts/4487/edit
http://staging-site-mojmir/admin/charts/4983/edit
http://staging-site-mojmir/admin/charts/486/edit
http://staging-site-mojmir/admin/charts/483/edit
http://staging-site-mojmir/admin/charts/4460/edit
http://staging-site-mojmir/admin/charts/4639/edit
http://staging-site-mojmir/admin/charts/4466/edit
http://staging-site-mojmir/admin/charts/6364/edit
http://staging-site-mojmir/admin/charts/4372/edit
http://staging-site-mojmir/admin/charts/4465/edit
http://staging-site-mojmir/admin/charts/3188/edit
http://staging-site-mojmir/admin/charts/4370/edit
http://staging-site-mojmir/admin/charts/5161/edit
http://staging-site-mojmir/admin/charts/4458/edit

There's a problem with missing **Data published by** for sources mentioned [above](https://github.com/owid/etl/pull/1528#issuecomment-1733741544). Do you know if this is a known issue? I could try fixing it in grapher and show both origins and sources.

**Comment by pabloarosado** [comment]
Hi @Marigold thanks for making the list. I have checked all of them and have the following comments:
* In all charts using per capita variables, I think the source reads well (sometimes you have to go to the "text" tab -> "Footer"  -> "Source" and click on the link button, to see the automatically generated source. Once you do that, you see the added "Various sources - Population (2023)". I will ask on slack if people agree with this way of quoting the population dataset, and whether we should always show it.
* I see that, when there are many sources, the footer automatically changes into "Multiple sources", which I think is a good approach (and probably with the new grapher redesign, this will look even better). This happens for example on [this chart](http://staging-site-mojmir/admin/charts/2028/edit). Although here maybe we have room for all sources, given that they are rather short. Maybe (**TODO in grapher**) we could make the logic a bit smarter, so that you get "Multiple sources" only if there's more than, say, 2 sources **and** we exceed a number of characters.
* It would be better if the "Various sources - Population (2023)" was always the last source listed. But I don't see an easy way to achieve this (do you?), so it's not a big deal.
* As you mentioned, the full citation is not properly combining sources and origins (**TODO in grapher**).
* I see that on [this chart](http://staging-site-mojmir/admin/charts/6935/edit), we see both the "Various sources - Population (2023)" and the individual population sources. This may be the only case where something seems to be wrong.

So, all in all, apart from the last small issue, it looks good!

**Comment by pabloarosado** [comment]
One additional point, @Marigold: As I mentioned on Slack, I'm not sure if scatter plots should also cite "Various sources - Population (2023)". Currently, I see that population is not cited anywhere on scatter plots (e.g. [this chart](http://staging-site-mojmir/admin/charts/4909/edit)). But I think we should probably cite population (no strong opinion). I suppose this is a **TODO in grapher** and may not affect this PR. What do you think?

**Comment by pabloarosado** [comment]
One minor change suggested by Ed (after asking Hannah and Max). On the `2023-03-31/population/population.meta.yml` file:
```
    attribution: Population based on various sources (2023)
```

**Comment by pabloarosado** [comment]
Hi @Marigold, the grapher issue of not combining sources and origins is probably affecting other charts. But after merging this PR, certainly hundreds of charts will be affected. So it would be good fix the grapher issue first. However, if you think it's easy to fix, we could merge this PR and fix the grapher issue soon after. I leave it up to you.

**Pull request merged**
